kratos/sources/kernel.cpp

```diff
@@ -59,6 +76,13 @@ namespace Kratos
         rOStream << std::endl;
         rOStream << "Conditions:" << std::endl;
         KratosComponents<Condition>().PrintData(rOStream);
+        
+        rOStream << "Loaded applications:" << std::endl;
+        
+        auto& application_list = Kernel::GetApplicationsList();
+        rOStream << "number of loaded applications = " << application_list.size() << std::endl;
+        for(auto it = application_list.begin(); it != application_list.end(); ++it)
+            rOStream << it->first << std::endl;
```

Comment:
cosmetic: but a leading "    " space would be better to see the application list!